sdk/python/lib/test/test_invoke.py

```diff
@@ -119,6 +119,34 @@ def test_invoke_empty_return(tok: str, version: str, empty: bool, expected) -> N
     assert pulumi.runtime.invoke(tok, props, opts).value == expected
 
 
+@pulumi.runtime.test
+def test_invoke_plain_invoke_takes_output_args() -> None:
```

Comment:
As with other tests, maybe a quick comment as to what it does? Or hoist out part of the comments within?

Reply:
There's a comment further down in the test, I moved it up to the top.